.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    branches: master
```

Comment:
Do we want this to be on master? Typically we would only publish to npm when a version is released, i.e. see https://github.com/crate/croud/blob/master/.github/workflows/release.yaml#L7

Reply:
The [action](https://github.com/marketplace/actions/npm-publish) that I have used in this workflow is automatically checking if the version has changed and it will publish it only if we are in this case. We can do this for every branch, but this means that we cannot run the `create_tag` script in this workflow (I was thinking about adding this part too)